demisto_sdk/__main__.py

```diff
@@ -191,16 +191,16 @@ def unify(**kwargs):
 @click.option(
     '--prev-ver', help='Previous branch or SHA1 commit to run checks against.')
 @click.option(
-    '--post-commit', is_flag=True, help='Whether the validation is done after you committed your files, '
-                                        'this will help the command to determine which files it should check in its '
-                                        'run. Before you commit the files it should not be used. Mostly for build '
-                                        'validations.')
+    '--post-commit', is_flag=True, help='Whether the validation should run only on committed branch changed files.'
```

Comment:
Maybe something like this is better, 
`'--post-commit', is_flag=True, help='Whether the validation should run only on the current branch's committed files.'`
but still - committed files that differ from master? Be explicit please.